simpledotorg/simple-server
# Update reporting facility appointment scheduled days

**Story card:** [sc-8282](https://app.shortcut.com/simpledotorg/story/8282/add-medication-dispensation-to-diabetic-patients-chart)

## Because

We need to show the days of medication dispensation for diabetic patients on the dashboard and in the facility downloads

## This addresses

Updates the following materialized views to calculate bucketed days to the next scheduled appointment
- reporting_facility_appointment_scheduled_days
- reporting_facility_states

## Test instructions
- Checkout this branch
- Run `db:migrate`
- You'll see diabetes specific columns in the above two view to calculate bucketed days to the next scheduled appointment
